src/client/components/dimension-tile/dimension-tile.tsx

```diff
@@ -233,6 +244,11 @@ export class DimensionTile extends React.Component<DimensionTileProps, Dimension
     this.fetchData(essence, timekeeper, dimension, sortOn, unfolded);
   }
 
+  private differentSortOn(oldSort: SortOn, newSort: SortOn): boolean {
```

Comment:
I would modify SortOn.equal static method instead of adding new one here

Reply:
Oooooooh, I forgot SortOn is not a record and has static methods ...